packages/mc-scripts/package.json

```diff
@@ -33,47 +33,47 @@
     ]
   },
   "dependencies": {
-    "@babel/core": "7.4.5",
+    "@babel/core": "7.5.5",
     "@commercetools-frontend/assets": "14.0.0",
     "@commercetools-frontend/babel-preset-mc-app": "14.0.0",
     "@commercetools-frontend/mc-dev-authentication": "14.0.0",
     "@commercetools-frontend/mc-html-template": "14.0.0",
-    "@svgr/webpack": "4.3.1",
+    "@svgr/webpack": "4.3.2",
     "babel-loader": "8.0.6",
     "babel-plugin-react-intl": "3.3.3",
-    "browserslist": "4.6.3",
+    "browserslist": "4.6.6",
     "clean-webpack-plugin": "3.0.0",
     "core-js": "3.1.4",
-    "css-loader": "3.0.0",
+    "css-loader": "3.1.0",
     "cssnano": "4.1.10",
-    "file-loader": "4.0.0",
+    "file-loader": "4.1.0",
     "fs-extra": "8.1.0",
     "glob": "7.1.4",
     "graphql-tag": "2.10.1",
     "html-webpack-plugin": "3.2.0",
     "json-loader": "0.5.7",
-    "mini-css-extract-plugin": "0.7.0",
+    "mini-css-extract-plugin": "0.8.0",
     "moment-locales-webpack-plugin": "1.0.7",
     "mri": "1.1.4",
     "optimize-css-assets-webpack-plugin": "5.0.3",
     "postcss": "7.0.17",
     "postcss-color-mod-function": "3.0.3",
     "postcss-custom-media": "7.0.8",
-    "postcss-custom-properties": "9.0.1",
+    "postcss-custom-properties": "9.0.2",
     "postcss-import": "12.0.1",
     "postcss-loader": "3.0.0",
-    "postcss-preset-env": "6.6.0",
+    "postcss-preset-env": "6.7.0",
     "postcss-reporter": "6.0.1",
     "postcss-safe-parser": "4.0.1",
     "react-dev-utils": "9.0.1",
-    "regenerator-runtime": "0.13.2",
+    "regenerator-runtime": "0.13.3",
```

Comment:
I am wondering if we actually still need this? /cc @emmenko

Reply:
Hmm no idea tbh...